src/test/java/org/testeditor/fixture/web/FirefoxTest.java

```diff
@@ -139,11 +141,16 @@ public void typeIntoTest() throws FixtureException {
         // given
```

Comment:
the name of the test is a bit too generic, don't you think?
how about testTypeIntoToReplaceExistingText?